requirements-py38.txt

```diff
@@ -401,6 +401,8 @@ werkzeug==2.0.2 \
     --hash=sha256:aa2bb6fc8dee8d6c504c0ac1e7f5f7dc5810a9903e793b6f715a9f015bdadb9a
     # via flask
 
-# WARNING: The following packages were not pinned, but pip requires them to be
-# pinned when the requirements file includes hashes. Consider using the --allow-unsafe flag.
-# setuptools
+# The following packages are considered to be unsafe in a requirements file:
```

Comment:
this can be removed.

Reply:
Sure, it's automatically generated but can be removed manually

Reply:
No longer needed, fix in new version